crates/metal-llvm/src/expr.rs

```diff
@@ -0,0 +1,78 @@
+use std::ffi::{c_uint, CString};
+
+use llvm_sys::{
+    core::{LLVMBuildCall2, LLVMConstInt, LLVMConstStringInContext, LLVMGetNamedFunction},
+    prelude::LLVMValueRef,
+};
+use metal_mir::{
+    expr::{literals::Literal, Expr},
+    parcel::Module,
+    types::primitives::Primitive,
+};
+
+use crate::{CodeGenType, CodeGenValue};
+
+fn args_to_values(llvm: &crate::LLVMRefs, module: &Module, args: &Vec<Expr>) -> Vec<LLVMValueRef> {
```

Comment:
[Should be `args: &[Expr]`](https://stackoverflow.com/a/40006220/16654863), not sure how i missed this earlier.